src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -54,6 +54,9 @@
     @Getter
     private static LogUnitServer logUnitServer;
 
+    @Getter
+    private static ManagementServer managementServer;
+
     public static boolean serverRunning_p = false;
```

Comment:
Hi, Medhavi.  That line was there before Zeeshan got to it.  It is my terrible style influence. I'm trying to correct my ways.  ^_^  But it will be a short while before cleanup is finished.  @zalokhan Feel free to change that name while you're there or leave it to me.